app/helpers/retina_image_helper.rb

```diff
@@ -17,6 +17,6 @@ def retina_image_tag(src, options={})
     src_2x = image_path(src.sub(/^(.*)(\.[^\.]+)$/, '\1@2x\2'))
     srcset = "#{src_1x} 1x,#{src_2x} 2x"
 
-    tag(:img, options.reverse_merge(src: src_1x, srcset: srcset, alt: ""))
+    tag(:img, **options.reverse_merge(src: src_1x, srcset: srcset, alt: ""))
```

Comment:
Ruby 3.0 stopped automatically splatting hash arguments into keyword arguments so we have to explicitly do it.